### PR TITLE
Documentation about page

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #1399 

Note: I have added a quick fix for the logos link.

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 9 min 14 sec

File: docs/src/user/manual/about.md [hunk 1]
```diff
@@ -1,2 +1,54 @@
-# About page
-<!-- To Be Populated -->
+# About SAWPS
+
+Welcome to the about page of the `South African Wildlife Population System (SAWPS)`. This page provides you with essential information about our platform, its purpose, and how to make the most of your experience here.
+
+## About Page
+![About Page](./img/about-page-1.png)
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/about.md:6]:
```suggestion
## About Page

![About Page](./img/about-page-1.png)
```

File: docs/src/user/manual/about.md [hunk 2]
```diff
@@ -1,2 +1,54 @@
...
+
+## Additional Resources and Pages
+
+![Additional Resources and Pages](./img/about-page-2.png)
+
+The footer section, located at the bottom of the `About Page`, offers links to additional resources and pages:
+
+1. **SANBI Logo**: By clicking on the `SANBI LOGO` you will navigate to the `SANBI (South African National Biodiversity Institute)` platform. The South African National Biodiversity Institute (SANBI) contributes to South Africa’s sustainable development by facilitating access to biodiversity data, generating information and knowledge, building capacity, providing policy advice, showcasing and conserving biodiversity in its national botanical and zoological gardens.
+
+    ### SANBI
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/about.md:26]:
```suggestion
    **SANBI**
```

File: docs/src/user/manual/about.md [hunk 3]
```diff
@@ -1,2 +1,54 @@
...
+
+    * **Map:** By choosing `MAP,` you can access geographical data visualisation and exploration features within our platform, also you can see the reports for the data.
+
+    * **Documentation:** Click `DOCUMENTATION` to access comprehensive documentation, like this page, which provides detailed information on various aspects of our platform.
+
+    * **Contact:** If you need to get in touch with us for any reason, the `CONTACT` link provides access to our support and communication channels.
+
+3. **IDS Logo**: By clicking on the `IDS LOGO` you will navigate to the `IDS (Information Decision Systems)` platform. Information Decision Systems (IDS) is a progressive Geographical Information Systems, Environmental Consulting and Disaster Management service provider.
+
+    ### IDS
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/about.md:42]:
```suggestion
    **IDS**
```

File: docs/src/user/manual/about.md [hunk 4]
```diff
@@ -1,2 +1,54 @@
...
+
+The footer section, located at the bottom of the `About Page`, offers links to additional resources and pages:
+
+1. **SANBI Logo**: By clicking on the `SANBI LOGO` you will navigate to the `SANBI (South African National Biodiversity Institute)` platform. The South African National Biodiversity Institute (SANBI) contributes to South Africa’s sustainable development by facilitating access to biodiversity data, generating information and knowledge, building capacity, providing policy advice, showcasing and conserving biodiversity in its national botanical and zoological gardens.
+
+    ### SANBI
+
+    ![SANBI](./img/about-page-3.png)
+
+2. **Additional Pages**: Usefull Pages links.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/about.md:30]:
```suggestion
2. **Additional Pages**: Useful Pages links.
```

File: docs/src/user/manual/about.md [hunk 5]
```diff
@@ -1,2 +1,54 @@
...
+
+3. **IDS Logo**: By clicking on the `IDS LOGO` you will navigate to the `IDS (Information Decision Systems)` platform. Information Decision Systems (IDS) is a progressive Geographical Information Systems, Environmental Consulting and Disaster Management service provider.
+
+    ### IDS
+
+    ![IDS](./img/about-page-4.png)
+
+4. **KARTOZA Logo**: By clicking on the `KARTOZA LOGO` you will navigate to the `KARTOZA` platform. Kartoza is a South Africa-based Free and Open Source GIS service provider. Kartoza develop and maintain geographic information systems and train teams to use geospatial software to its full potential.
+
+    ### KARTOZA
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/about.md:48]:
```suggestion
    **KARTOZA**
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 5 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 22 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 8 min 36 sec

**Comment by dimasciput** [comment]
@Jeremy-Prior please review

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 7 min 30 sec

**Pull request merged**